src/components/procedure/_remove-demo-project.mdx

```diff
@@ -0,0 +1,6 @@
+To remove a demo project, follow these steps:
+
+1. In Semgrep App, go to the **[Projects](https://semgrep.dev/orgs/-/projects)** page.
+1. Click the <i class="fa-solid fa-gear"></i> **gear** icon of the **Demo project - OWASP Juice Shop**.
+1. Click the <i class="fa-regular fa-ellipsis"></i> ellipsis icon, and then click <i class="fa-sharp fa-solid fa-xmark"></i> **Delete project**.
+    ![Screenshot of Semgrep App Projects page with the **Delete project** button.](/img/semgrep-app-delete-demo-project.png "Screenshot of Semgrep App Projects page with the **Delete project** button.")
```

Comment:
This works great, but I think another way to remove a project is to check the checkbox on the left and hit delete. You might have chosen just one workflow to not confuse users. If so, please ignore this comment 😅

Reply:
Ah true! I did not notice, easier procedure, I will change it. Thanks. :)

Reply:
@pangngan please check again, added!

![Screenshot 2022-12-13 at 20 09 45](https://user-images.githubusercontent.com/26669307/207423594-e03480d5-affd-4142-8838-d9918f58b3b7.png)

Reply:
Looks great to me! :D